locksmith/src/controllers/userController.ts

```diff
@@ -1,9 +1,13 @@
-import UserOperations = require('../operations/userOperations')
-import { Request, Response } from 'express-serve-static-core'
+import { Request, Response } from 'express-serve-static-core' // eslint-disable-line no-unused-vars, import/no-unresolved
 import { DecoyUser } from '../utils/decoyUser'
 
+import UserOperations = require('../operations/userOperations')
+
 namespace UserController {
-  export async function createUser(req: Request, res: Response): Promise<any> {
+  export const createUser = async (
```

Comment:
Do you have a link to any documentation around with function definitions in this fashion are preferred?

Reply:
This is one example of eslint doing the wrong thing. This is totally valid TS, but eslint doesn't like having a function declaration at anything other than the program root. Creating the functions as a value is allowed everywhere, so this should have the same exact behavior while causing eslint to be quiet.

On a side note, doing things inside namespaces may not be necessary with ES6 modules, but we can concern ourselves with that at a later date.